message MSG_CLIENT_DM_DOMAIN_DESCRIPTION_UPDATE
{
    optional string token = 1;
    optional string domainName = 2;
    optional string domainDescription = 3;
}

message MSG_DM_CLIENT_DOMAIN_DESCRIPTION_UPDATE_ACK
{
    optional int32 statusCode = 1;
}